packages/flame_steering_behaviors/example/README.md

```diff
@@ -0,0 +1,14 @@
+# Example
```

Comment:
```suggestion
# Flame Steering Behaviors' example
```

Reply:
Same as with the pubspec name, keeping it as is to keep it in-line with the `flame_behaviors` example